Simplify the prompting for a new sui config in the CLI (#24627)
MystenLabs/sui: docs/content/guides/developer/getting-started/configure-sui-client.mdx

## `client.yaml`

Your Sui client is now configured. By default, Sui stores this information in either the `~/.sui/sui_config/client.yaml` file (macOS/Linux) or `%USERPROFILE%\.sui\sui_config\client.yaml` file (Windows). You can store a `client.yaml` file in a different location, if preferred, and specify its location with the `--client.config` flag.

The `client.yaml` contains the configuration for connecting to different Sui networks (testnet, mainnet, devnet, and localnet), as well as your current "active environment", which is tells the CLI which network to use if you don't explicitly provide one.

> You can modify your configuration (for example by adding new RPC endpoints or changing your active environment) using the `sui client` subtools; see `sui client --help` for more information.

Reviewer: ```suggestion
You can modify your configuration using the `sui client` subcommands. See the output of `sui client --help` for more information.
```
